cli/pruning/pruning.go

```diff
@@ -86,6 +90,14 @@ func NewCommand() *cli.Command {
 				Name:  "pruning",
 				Usage: "prune state trie",
 			},
+			cli.BoolFlag{
+				Name:  "seq",
+				Usage: "prune state trie sequential mode",
+			},
+			cli.BoolFlag{
+				Name:  "lowmem",
```

Comment:
What about lowram?

Reply:
hm, memory refers to ram, and storage refers to disk.